asciidoc-confluence-publisher-doc/etc/docs/00-index.adoc

```diff
@@ -249,6 +258,7 @@ docker run --rm -e ROOT_CONFLUENCE_URL=http://confluence-host \
    -e PAGE_TITLE_SUFFIX=" (V 1.0)" \
    -e PUBLISHING_STRATEGY=REPLACE_ANCESTOR \
    -e VERSION_MESSAGE=V 1.0 \
+   -e ATTRIBUTES={"attribute1": "value1", "attribute2": "value2"} \
```

Comment:
This actually does not work, since the JSON string is not properly put into quotes. And when putting the whole string into double quotes, It only works either of the two following ways:

`-e ATTRIBUTES="{\"attribute1\": \"value1\", \"attribute2\": \"value2\"}" \` (double quotes + escaping)
`-e ATTRIBUTES='{"attribute1": "value1", "attribute2": "value2"}' \` (single quotes, while loosing shell variable replacements)

That was one of the remarks regarding inconvenience I had when we were discussing the options for providing the user attributes to the Docker container. Anyway, as long as we have a working example, that should be ok for now. Maybe the version with the single quotes is more readable.

Reply:
One more thing I have noted: user attributes are currently only getting replaced in the content of a page, but not in the page title. 

We should either document this limitation or enhance (and test) the support accordingly. I have not yet investigated in detail, but I guess it is related to the way we are retrieving the documents title via AsciiDoc (we are not reading it from the converted content, but using `ASCIIDOCTOR.readDocumentHeader(...)`).